src/components/About/Blurbs.js

```diff
@@ -0,0 +1,199 @@
+          </Text>
+          <View style={styles.leftCircleView}>
+            <Image source={require("../../img/img-clipart-os-jam.png")} style={styles.leftBlurbImages}/>
+          </View>
+        </View>
+
+        <View style={styles.blurbsView}>
+           <View style={styles.rightCircleView}>
+            <Image source={require("../../img/img-logo-colored.jpg")} style={styles.fillLogoBlurb}/>
+          </View>
+          <Text style={styles.blurbs}>
+            And that''s the story of how fortissimo was conceived.{"\n"}
+          </Text>
+        </View>
+
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+    verticalAlign: 'middle',
+    margin: 20,
+  },
+
+  header: {
+    color: '#9739E8',
+    fontSize: 35,
+    fontStyle: 'italic',
+    fontWeight: 'lighter',
+    textAlign: 'center',
+  },
+
+  blurbsView: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    paddingTop: 20,
+    paddingBottom: 20,
+  },
+
+  blurbs: {
+    position: 'relative',
+    left: '5%',
+    backgroundColor: '#E8D3F7',
+    width: '73%',
+    fontFamily: 'Raleway',
+    fontSize: 8,
+    textAlign: 'center',
+    fontWeight: 'lighter',
+    padding: 20,
+    borderRadius: 5,
+    display: 'inline-block',
+    overflow: 'hidden',
+  },
+
+  leftCircleView: {
+    width: 80,
+    height: 80,
+    display: 'flex',
+    flexDirection: 'row',
+    padding: 10,
+    borderRadius: 50,
+    marginLeft: 20,
+    backgroundColor: 'white',
+
+    shadowColor: '#000000',
+    shadowOffset: {
+      width: 0,
+      height: 0,
+    },
+    shadowRadius: 3,
+    shadowOpacity: 0.5,
+  },
+
+  rightCircleView: {
+    width: 80,
+    height: 80,
+    display: 'flex',
+    flexDirection: 'row',
+    padding: 0,
+    borderRadius: 50,
+    marginRight: 10,
+    backgroundColor: 'white',
+
+    shadowColor: '#000000',
+    shadowOffset: {
+      width: 0,
+      height: 0,
+    },
+    shadowRadius: 3,
+    shadowOpacity: 0.5,
+  },
+
+  leftBlurbImages: {
+    width: 50,
+    height: 50,
+    marginTop: 'auto',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    backgroundColor: 'white',
+    resizeMode: 'contain',
+  },
+
+  rightBlurbImages: {
+    width: 50,
+    height: 50,
+    marginTop: 'auto',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    backgroundColor: 'white',
+    resizeMode: 'contain',
+  },
+
+  fillSfImage: {
+    width: 80,
+    height: 80,
+    marginTop: -10,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    borderRadius: 40,
+    backgroundColor: 'white',
+  },
+
+  fillLogoBlurb: {
+    width: 80,
+    height: 80,
+    marginTop: 'auto',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    borderRadius: 40,
+    backgroundColor: 'white',
+  }
+
+});
```

Comment:
About/Blurbs.js: Code is clean and well-organized for future additions/amendments.